tools/ghidra/emit-artifact-index.py

```
@@ -0,0 +1,127 @@
+            if isinstance(raw, dict):
+                fingerprint_id = str(raw.get("fingerprintId", "")).strip()
+                module_name = str(raw.get("moduleName", "")).strip()
+                file_sha256 = str(raw.get("fileSha256", "")).strip()
+                if fingerprint_id and module_name and file_sha256:
+                    return {
+                        "fingerprintId": fingerprint_id,
+                        "moduleName": module_name,
+                        "fileSha256": file_sha256,
+                    }
+        except Exception:
+            pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)